qdax/core/emitters/multi_emitter.py

```diff
@@ -0,0 +1,87 @@
+from typing import Optional, Tuple
+
+import jax
+from jax import numpy as jnp
+
+from qdax.core.containers.repertoire import Repertoire
+from qdax.core.emitters.emitter import Emitter, EmitterState
+from qdax.types import Descriptor, ExtraScores, Fitness, Genotype, RNGKey
+
+
+class MultiEmitterState(EmitterState):
+    emitters_state_tuple: Tuple[EmitterState, ...]
+
+
+class MultiEmitter(Emitter):
+    def __init__(self, emitters_tuple: Tuple[Emitter, ...]):
+        self.emitters_tuple = emitters_tuple
+
+    def emit(
+        self,
+        repertoire: Optional[Repertoire],
+        multi_emitter_state: Optional[MultiEmitterState],
+        random_key: RNGKey,
+    ) -> Tuple[Genotype, RNGKey]:
+        assert multi_emitter_state is not None
+        assert len(multi_emitter_state.emitters_state_tuple) == len(self.emitters_tuple)
+
+        random_key, *_keys_emitters = jax.random.split(
+            random_key, len(self.emitters_tuple) + 1
+        )
+
+        all_genotypes = []
+        for emitter, emitter_state, _key_emitter in zip(
+            self.emitters_tuple,
+            multi_emitter_state.emitters_state_tuple,
+            _keys_emitters,
+        ):
+            genotype, _ = emitter.emit(repertoire, emitter_state, _key_emitter)
+            all_genotypes.append(genotype)
+
+        genotypes_tree = jax.tree_map(
```

Comment:
[Naming] and replace `genotypes_tree` by `offspings`??

Reply:
agreed!